src/common/download.js

```diff
@@ -11,6 +16,11 @@ export function downloadUrl(url, name, callback) {
   }, 3000);
 }
 
+/**
+ * @param {object} blob
```

Comment:
While not a mistake but we already use `Object` so it should be used everywhere consistency.

Reply:
Also, it should be `Blob`, not `Object`.